test/distributed/org/apache/cassandra/distributed/fuzz/HarryRunner.java

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.distributed.fuzz;
+
+import java.io.File;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import harry.core.Configuration;
+import harry.ddl.SchemaSpec;
+import harry.model.sut.SystemUnderTest;
+import harry.runner.Runner;
+
+public class HarryRunner
+{
+    public static final SchemaSpec schemaSpec = null;
+
+    public static void init()
+    {
+        InJvmSut.init();
+        PrintlnSutConfiguration.init();
+        FaultInjectingPartitionVisitor.init();
+        QueryingNoOpChecker.init();
+        Configuration.registerSubtypes(Configuration.NoOpMetricReporterConfiguration.class);
+        Configuration.registerSubtypes(Configuration.RecentPartitionsValidatorConfiguration.class);
+    }
+
+    public static void main(String[] args) throws Throwable
+    {
+        System.setProperty("log4j2.disableJmx", "true"); // setting both ways as changes between versions
+        System.setProperty("log4j2.disable.jmx", "true");
+        System.setProperty("log4j.shutdownHookEnabled", "false");
+        System.setProperty("cassandra.allow_simplestrategy", "true"); // makes easier to share OSS tests without RF limits
+        System.setProperty("cassandra.minimum_replication_factor", "0"); // makes easier to share OSS tests without RF limits
+
+        System.setProperty("cassandra.disable_tcactive_openssl", "true");
+        System.setProperty("relocated.shaded.io.netty.transport.noNative", "true");
+        System.setProperty("org.apache.cassandra.disable_mbean_registration", "true");
+
+        // Initialize YAML subtypes
+        init();
+
+        harry.runner.HarryRunner runner = new harry.runner.HarryRunner()
+        {
+            public void beforeRun(Runner.TimedRunner runner)
+            {
+                runner.getRun().sut.schemaChange("CREATE KEYSPACE " + schemaSpec.keyspace + " WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 2};");
```

Comment:
Is it possible for `schemaSpec` to be anything other than `null` here?

Reply:
Expanding on this, if you try to run the `main()` method, the `schemaSpec` is indeed `null`, and you get a NPE when `createRun()` tries to create a `SchemeSpec` from the `SchemaProviderConfiguration`...

```
Exception in thread "main" java.lang.NullPointerException
	at harry.core.Configuration.createRun(Configuration.java:238)
	at harry.core.Configuration.createRunner(Configuration.java:255)
	at harry.core.Configuration.createRunner(Configuration.java:213)
	at harry.runner.HarryRunner.run(HarryRunner.java:63)
	at org.apache.cassandra.distributed.fuzz.HarryRunner.main(HarryRunner.java:71)
```

Reply:
removed harry runner from tree for now